res/http.php

```diff
@@ -109,7 +110,12 @@
             $emails = $emailRepository->findLatest($since);
             return $rss->build($emails);
         },
-
+        '/rss-rfc' => function (EmailRepository $emailRepository, ServerRequestInterface $request) {
+            newrelic_name_transaction('rss-rfc');
+            $rss = new RssRfcBuilder($request);
+            $threads = $emailRepository->findThreads("where threadInfos.subject like '%RFC%'", 'order by threadInfos.date desc', 1, null);
```

Comment:
Oh I think the `findThreads()` method is public by mistake (only called in `EmailRepository`). Could you make it private and add a new method like `$emailRepository->findRfcThreads()` (like `findTopThreads()` for example)?

Reply:
I've moved the method in the repository and made the other one private.